src/core/lib/iomgr/timer_generic.cc

```diff
@@ -97,6 +97,12 @@ static void init_timer_ht() {
   }
 }
 
+static void destroy_timer_ht() {
+  for (int i = 0; i < NUM_HASH_BUCKETS; i++) {
```

Comment:
Use range for loop?

Reply:
It is not clear to me in which places it is Ok to use `C++11` constructs, so trying to avoid it and not scare people :)